recipes/download.rb

```diff
@@ -92,3 +92,13 @@
   owner 'root'
   group 'root'
 end
+
+s3_file node['nexus_repository_manager']['nexus_data']['etc']['path'] + '/license.lic' do
```

Comment:
Where was this placed before we made it conditional?

Reply:
this is now conditional, see the 'not_if' condition a few lines down.